src/components/Community/Contribute/index.tsx

```diff
@@ -29,93 +31,87 @@ export default function CommunityContribute({ theme }) {
         mobileDescription={mobileDescription}
         title={title}
       >
-        <Items>
-          <Item>
+        <div className={sharedStyles.items}>
+          <div className={sharedStyles.item}>
             <CommunityBlock
               title="Make a PR"
               action={
-                <Button
-                  theme={theme}
+                <Link
+                  className={sharedStyles.button}
+                  style={theme}
                   href="https://github.com/iterative/dvc"
                   target="_blank"
-                  rel="noreferrer noopener"
                   onClick={logPR}
                 >
                   Go to Github
-                </Button>
+                </Link>
               }
             >
               Let&apos;s build something great together. Become a DVC
               contributor!.
             </CommunityBlock>
-          </Item>
-          <Item>
+          </div>
+          <div className={sharedStyles.item}>
             <CommunityBlock
               title="Write a blogpost"
               action={
-                <Button
-                  theme={theme}
+                <Link
+                  className={sharedStyles.button}
+                  style={theme}
                   href="mailto:info@dvc.org?subject=I want to write a blogpost!"
                   target="_blank"
-                  rel="noreferrer noopener"
                   onClick={logBlogpost}
                 >
                   Let’s talk!
-                </Button>
+                </Link>
               }
             >
               We&apos;re always interested in guest writers for our blog. If you
               have something to share, please reach out!
             </CommunityBlock>
-          </Item>
-          <Item>
+          </div>
+          <div className={sharedStyles.item}>
             <CommunityBlock
               title="Give a talk"
               action={
-                <Button
-                  theme={theme}
+                <Link
+                  className={sharedStyles.button}
+                  style={theme}
                   href="mailto:info@dvc.org?subject=I want to give a talk!"
                   target="_blank"
-                  rel="noreferrer noopener"
```

Comment:
Why remove this?

Reply:
It's now automatically handled by Link component